shopping-cart/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+    <link rel="stylesheet" href="shopping-cart.css">
+    <link rel="stylesheet" href="../common/main.css">
+</head>
+
+<body>
+    <h3>Shopping Cart</h3>
+    <table>
+        <thead>
+            <tr>
+                <th class="align-left">Cues</th>
+                <th>Qty</th>
+                <th>Price</th>
+                <th>Total</th>
+            </tr>
+        </thead>
+        <tbody>
+        </tbody>
+        <tfoot>
+            <tr>
+                <th></th>
+                <th></th>
+                <th>Order Total:</th>
+                <th id="order-total-cell"></th>
+            </tr>
+        </tfoot>
+    </table>
+    <script src="./app.js" type="module"></script>
+    <script src="./render-table-row.js" type="module"></script>
```

Comment:
you only need to import app.js because its a module import it can grab anything else it needs through app.js file.